analyzer/src/main/kotlin/managers/Bundler.kt

```diff
@@ -175,26 +174,28 @@ class Bundler : PackageManager() {
                 it?.merge(localGemSpec) ?: localGemSpec
             }
 
+            val gemId = Identifier(javaClass.simpleName, "", gemSpec.name, gemSpec.version)
```

Comment:
Just use `toString()` instead of `javaClass.simpleName`.

Reply:
Done. I also reused the `Identifier` object where the `PackageReference` is added further down in the method.